build.gradle

```diff
@@ -57,13 +57,13 @@ allprojects {
     checkstyleToolVersion = '8.30'
     // Both `patchPluginXml.untilBuild` in root project config and `intellij_versions` in scripts/verify-binary-compatibility
     // must also be updated when we update the version below.
-    intellijVersion = '2020.1'
-    jgitVersion = '5.5.1.201910021850-r'
+    intellijVersion = '2020.1.1'
```

Comment:
`Dockerfile` rebuild needed, right?

Reply:
Yes, would be useful after this PR is merged

Reply:
Or actually, let's wait for #265